app/events/db/db_table_field_mappings.yaml

```diff
@@ -608,3 +608,43 @@ mappings:
             profile_following_count: app.events.db.value_post_processors.cast_string_to_int
             profile_subscriber_count: app.events.db.value_post_processors.cast_string_to_int
             contact_details: app.events.db.value_post_processors.cast_to_json
+
+
+    profile_analytics:
```

Comment:
Check: Getting an error in Pycharm here on duplicate name. Working fine via Postman.